be/src/main/java/kr/codesquad/sidedish/service/ShippingInfoDTO.java

```diff
@@ -9,15 +9,16 @@
 @Getter
 public class ShippingInfoDTO {
 
-	private final Integer deliveryType;
+	private final Integer deliveryForm;
 	private final Integer deliveryCharge;
 	private final String deliveryInfo;
 
 	public static ShippingInfoDTO from(ShippingInfo shippingInfo) {
 
 		DeliveryType deliveryType = DeliveryType.convertInfo(shippingInfo.getDeliveryInfo());
 
-		return new ShippingInfoDTO(shippingInfo.getDeliveryType(), shippingInfo.getDeliveryCharge(),
+		return new ShippingInfoDTO(shippingInfo.getDeliveryForm(),
+			shippingInfo.getDeliveryCharge(),
 			deliveryType.getValue());
```

Comment:
```suggestion
		return new ShippingInfoDTO(
		        shippingInfo.getDeliveryForm(),
			shippingInfo.getDeliveryCharge(),
			deliveryType.getValue()
		);
```

`nit` 보통 가독성을 위해 이렇게 만드는 것 같아요.